spring-core/src/main/java/org/springframework/core/annotation/AnnotationTypeMappings.java

```diff
@@ -91,9 +91,6 @@ private void addMetaAnnotationsToQueue(Deque<AnnotationTypeMapping> queue, Annot
 					.findRepeatedAnnotations(metaAnnotation);
 			if (repeatedAnnotations != null) {
 				for (Annotation repeatedAnnotation : repeatedAnnotations) {
-					if (!isMappable(source, metaAnnotation)) {
```

Comment:
I'm wondering if that was perhaps a copy-and-paste error and if it should rather be:

```java
if (!isMappable(source, repeatedAnnotation)) {
    continue;
}
```

@philwebb, what are your thoughts on the matter?

Reply:
@philwebb, I'm assuming my above assumption is correct. (I know: a lot of assuming).

In light of that, we'll go forward with the change in this PR.

Reply:
I think it was a typo. A little concerning that no test covered it.